flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -1111,6 +1111,53 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       math.atan(-0.1231231321321321111).toString)
   }
 
+  @Test
+  def testAtan2(): Unit = {
+    testAllApis(
+      'f25.atan2('f26),
+      "f25.atan2(f26)",
+      "ATAN2(f25, f26)",
+      math.atan2(0.42.toByte, 0.toByte).toString)
+
+
```

Comment:
Remove this blank line would be better~

Reply:
Thank you for that catch. 
blank line removed